### DISCUSSION TITLE
keyboard not working on rustdesk 1.3.0 on macOS 15 connecting to windows rustdesks

### DISCUSSION THREAD

**Discussion by paulpalaszewski** [rustdesk/rustdesk#9470] [General]
### Bug Description

For about two weeks now I'm having troubles with rustdesk on macOS: It connects, I get the live-screen, can move mouse + click on things, but I can't type. Any keyboard input I do does not reach the other side.

### How to Reproduce

Things I did / tried ...
- Updated rustdesk 1.2.6 to 1.3.0. I'm not 100% sure, if the issue started with 1.2.6 oder right after the update or if keyboard inputs worked for a few days after the update
- Restarting rustdesk or rebooting macOS did not help
- Downloaded rustdesk 1.2.6 again, issue remained
- Updated to macOS 15.0, issue remained
- Tried rustdesk 1.3.0 on Windows 11 - no issues

### Expected Behavior

Keyboard works.

### Operating system(s) on local side and remote side

macOS 15.0 -> Windows 10+11

### RustDesk Version(s) on local side and remote side

1.3.0 + 1.2.6 -> 1.2.6 + 1.3.0

### Screenshots

not applicable

### Additional Context

Any ideas / help appreciated!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3261#issuecomment-1768000251

**Reply by rustdesk** [reply]
or change input source
![image](https://github.com/user-attachments/assets/1ed48a0a-431c-4e77-bf54-875a9a13106c)
